packages/core/src/components.d.ts

```diff
@@ -333,7 +343,7 @@ export namespace Components {
         /**
           * The status of the Badge.
          */
-        "status"?: "primary" | "success" | "info" | "warning" | "danger" | "error" | "outline" | "attention";
+        "status"?: BadgeStatusVariant;
```

Comment:
Ik denk dat `BadgeStatusVariant` dubbelop is. De varianten van een Badge worden uitgedrukt in "statussen".

Zullen we er `BadgeStatus` van maken? Past ook beter in de API:

```html
<dso-badge status="...">
```

Reply:
Aangepast